UnityProject/Assets/Scripts/Lighting/LightMountStates.cs

```diff
@@ -87,6 +87,12 @@ public bool WillInteract(HandApply interaction, NetworkSide side)
 
 	public void ServerPerformInteraction(HandApply interaction)
 	{
+		if (!isServer) 
```

Comment:
ServerPerformInteraction is only ever called on server via Interface. `!isServer` is not needed here

Reply:
Also completely agree with this.